From 2d0ae7baf83e5e14256837c8f18893035b86ac15 Mon Sep 17 00:00:00 2001
From: shengfei Xu <xsf@rock-chips.com>
Date: Tue, 17 Apr 2018 11:05:30 +0800
Subject: [PATCH] arm64: dts: rockchip: update opp-table for rk3308

Analysis the data of different leakage, if the frequency is 1.2G,
then the voltage must be 1200mV.

Change-Id: I5160cf2ec7bab48172f549b138881656b9538cd3
Signed-off-by: shengfei Xu <xsf@rock-chips.com>
---
 arch/arm64/boot/dts/rockchip/rk3308.dtsi | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm64/boot/dts/rockchip/rk3308.dtsi b/arch/arm64/boot/dts/rockchip/rk3308.dtsi
index 43ca2f20e07c..d04cbf3b7c70 100644
--- a/arch/arm64/boot/dts/rockchip/rk3308.dtsi
+++ b/arch/arm64/boot/dts/rockchip/rk3308.dtsi
@@ -103,7 +103,7 @@
 		};
 		opp-1200000000 {
 			opp-hz = /bits/ 64 <1200000000>;
-			opp-microvolt = <1175000 1175000 1340000>;
+			opp-microvolt = <1200000 1200000 1340000>;
 			clock-latency-ns = <40000>;
 		};
 	};
-- 
2.35.3

